src/Backend.cpp

```diff
@@ -188,6 +189,19 @@ std::vector<std::shared_ptr<HAEntity>> HABackend::GetEntitiesByDomain(const std:
   return ret;
 }
 
+std::vector<std::shared_ptr<HAEntity>> HABackend::GetEntitiesByPattern(const std::string& pattern)
```

Comment:
```suggestion
// FIXME: a glob is likely to be more natural for users
std::vector<std::shared_ptr<HAEntity>> HABackend::GetEntitiesByPattern(const std::string& pattern)
```

Reply:
alternatively, we demand `//` around regex for now so we can add glob (without `//`) later